tools/taskcluster-worker-runner/provider/aws/awsprovider.go

```diff
@@ -58,7 +58,7 @@ func (p *AWSProvider) ConfigureRun(state *run.State) error {
 		"signature": interface{}(instanceIdentityDocumentSignature),
 	}
 
-	err = provider.RegisterWorker(
```

Comment:
A comment here indicating that this temporarily ignores the config would be good (pointing to this bug # for followup?)